flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlAlterConnectionReset.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.sql.parser.ddl;
+
+import org.apache.flink.sql.parser.SqlParseUtils;
+import org.apache.flink.sql.parser.SqlUnparseUtils;
+
+import org.apache.calcite.sql.SqlIdentifier;
+import org.apache.calcite.sql.SqlNode;
+import org.apache.calcite.sql.SqlNodeList;
+import org.apache.calcite.sql.SqlWriter;
+import org.apache.calcite.sql.parser.SqlParserPos;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * ALTER CONNECTION [IF EXISTS] [[catalogName.] dataBasesName.]connectionName RESET ( 'key1' [,
+ * 'key2']...).
+ */
+public class SqlAlterConnectionReset extends SqlAlterConnection {
+    private final SqlNodeList optionKeyList;
+
+    public SqlAlterConnectionReset(
+            SqlParserPos pos,
+            SqlIdentifier connectionName,
+            boolean ifConnectionExists,
+            SqlNodeList optionKeyList) {
+        super(pos, connectionName, ifConnectionExists);
+        this.optionKeyList = requireNonNull(optionKeyList, "optionKeyList should not be null");
+    }
+
+    @Override
+    public List<SqlNode> getOperandList() {
+        return List.of(name, optionKeyList);
+    }
+
+    public Set<String> getResetKeys() {
+        return optionKeyList.getList().stream()
+                .map(SqlParseUtils::extractString)
+                .collect(Collectors.toSet());
+    }
```

Comment:
This is not used

in case we really need it, how about creating a method in `SqlParseUtils` somcething like `extractList`?
We already have there `extractMap` and `extractList` https://github.com/apache/flink/blob/d0c9ed9ff47cd0f0fae62958521a0b18e5cd9bf3/flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/SqlParseUtils.java#L83-L99